index.html

```diff
@@ -273,8 +277,8 @@ <h3 class="feature-title">Contribute</h3>
                                         </svg>
                                     </div>
                                     <h3 class="feature-title">Support</h3>
-                                    <p class="text-sm">We're always wanting to add new spoken and programming languages to our repository. Please lege-<u>share</u> our mission.
-                                    </br></br>From beginners learning how to code to collaborating with diverse teams around the world, Legesher connects us all.</p>
+                                    <p class="text-sm">We always want to add new spoken and programming languages to our repository. Please lege-<u>share</u> our mission.
+                                    </br></br>From beginners to professionals, learning how to code to collaborating with diverse teams around the world, Legesher connects us all.</p>
```

Comment:
You're on the right track, let's refine this a little more. 

The first <p> sentence's purpose is to encourage new dialect translations to be added / reviewed. 
The second sentence is to showcase the different use cases of the tool: 
- people just starting to learn how to code 
- professional developers collaborating in (diverse) teams